### PR TITLE
Create inference.py

### PR DESCRIPTION
**Pull request by hassaanQadir**
This is script 2 out of 2 accompanying my 30Aug24 Fine-Tuning Llama3 Article.

### CODE REVIEW

File: finetuning/llama/inference.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# inference.py
```
Comment (mernit) [inline finetuning/llama/inference.py:1]: add comment that shows command to deploy to beam

**Review by mernit — CHANGES_REQUESTED** [review]

File: finetuning/llama/inference.py (hunk 1 above)
Comment (hassaanQadir) [reply finetuning/llama/inference.py:1]: I added 

"Deploy to beam by running `$ python inference.py` in the terminal"

**Review by hassaanQadir — COMMENTED** [review]
I added 

"Deploy to beam by running `$ python inference.py` in the terminal"

**Comment by mernit** [comment]
> I added
> 
> "Deploy to beam by running `$ python inference.py` in the terminal"

This is not the correct command, format is `beam deploy file.py:handler_func`

**Review by hassaanQadir — COMMENTED** [review]
I added

 Deploy to beam by running `$ beam deploy inference.py:predict` in the terminal

**Review by mernit — APPROVED** [review]

**Pull request merged**